### PR TITLE
Added in Tutor, Student, Course Schema

### PR DESCRIPTION
**Pull request by heidage**
- Please help me check if these are the relations we want, I referenced the one we have on Figma

### CODE REVIEW

File: Bamboo/backend/bamboo-backend/prisma/schema.prisma [hunk 1]
```diff
@@ -13,6 +13,114 @@ datasource db {
...
+  tutors Tutor[] @relation("TutorCourses")
+  AdminId String
+  courseAdmin Tutor @relation("AdminCourses",fields: [AdminId], references: [id])
+  students Student[]
+  studentGroups StudentGroup[]
+  announcements Announcements[]
+  description Description[]
+  topics Topic[]
+  assignments Assignment[]
+}
```
Comment (TsienJin) [inline Bamboo/backend/bamboo-backend/prisma/schema.prisma:62]: Can we add a created and updated on field?

File: Bamboo/backend/bamboo-backend/prisma/schema.prisma [hunk 2]
```diff
@@ -13,6 +13,114 @@ datasource db {
...
+
+model Description {
+  id String @id @default(cuid())
+  summary String @db.VarChar(255)
+  subjectDetails String @db.VarChar(255)
+  courseId String
+  course Course @relation(fields: [courseId], references: [id])
+}
+
+model Topic {
```
Comment (TsienJin) [inline Bamboo/backend/bamboo-backend/prisma/schema.prisma:81]: Are we adding in self relations for subtopics?

File: Bamboo/backend/bamboo-backend/prisma/schema.prisma [hunk 3]
```diff
@@ -13,6 +13,114 @@ datasource db {
...
+  description String @db.VarChar(255)
+  courseId String
+  topicId String
+  topic Topic @relation(fields: [topicId], references: [id])
+  videoContent String?
+  documentFile String
+  richText String
+}
+
+model Assignment {
```
Comment (TsienJin) [inline Bamboo/backend/bamboo-backend/prisma/schema.prisma:105]: Should we add in relation for studentGroups? So this way when a student is added into a student group, they will have access to the assignment

**Review by TsienJin — COMMENTED** [review]
Added some comments, lmk what u think!

**Comment by heidage** [comment]
Added in self-relation for topics, created and updated on date for courses and studentGroup relation for Assignments

**Comment by TsienJin** [comment]
LGTM

**Pull request merged**